chain/network/src/network_protocol/tests.rs

```diff
@@ -170,3 +176,49 @@ fn serialize_deserialize() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn sign_and_verify_with_signed_ip_address_interface() {
+    // Design sign and verify algorithm for std::net::IpAddr
```

Comment:
Don't understand this comment; is it here by mistake?

Reply:
ah my bad, outdated comment. removed. thanks